crates/test-framework/src/spicetest/vector_search/worker.rs

```diff
@@ -120,13 +128,23 @@ pub(crate) struct VectorSearchWorker {
 }
 
 #[derive(Deserialize)]
-pub(crate) struct Match {
-    score: f64,
+#[allow(dead_code)]
+pub(crate) struct SearchResponse {
+    results: Vec<SearchResponseResult>,
+    duration_ms: Option<u64>,
 }
 
 #[derive(Deserialize)]
-pub(crate) struct SearchResponse {
-    matches: Vec<Match>,
+#[allow(dead_code)]
+pub(crate) struct SearchResponseResult {
```

Comment:
Note, we can re-use SearchResponse definition from Runtime crate, but this will dramatically increase build time for test operator. Thus, keeping own simple definition for SearchResult